src/Microsoft.AspNetCore.Blazor.Server/Microsoft.AspNetCore.Blazor.Server.csproj

```diff
@@ -1,10 +1,18 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\Microsoft.AspNetCore.Blazor.Browser\Hosting\ConventionBasedStartup.cs" Link="Hosting\ConventionBasedStartup.cs" />
+    <Compile Include="..\Microsoft.AspNetCore.Blazor.Browser\Hosting\IBlazorStartup.cs" Link="Hosting\IBlazorStartup.cs" />
+    <Compile Include="..\Microsoft.AspNetCore.Blazor.Browser\Services\BrowserUriHelperInterop.cs" Link="Circuits\BrowserUriHelperInterop.cs" />
```

Comment:
Since this project references `Microsoft.AspNetCore.Blazor.Browser` anyway, should we use `[InternalsVisibleTo]` to avoid the need for these special shared sources?

Reply:
I think this is roughly the same lvel of complexity. If we're going to start using IVT I would have designed a few other things differently 😆